docs/getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started
+
+Welcome to the Getting Started guide for the refarch-templates. This guide
+is divided into sections for Frontend/Web Component and Backend/EAI, as well
+as a general setup and requirements applicable to all templates.
+
+## General
+
+To use the refarch-templates, you need to have Docker installed on your
+machine. Inside the `stack` folder, you will find a `docker-compose.yml`
+file that will spin up everything needed for local development. 
+
+When developing locally, the stack should always be running. 
+### Download Templates
+
+You can download the templates or clone them directly from the GitHub
```

Comment:
I don't think we should recommend cloning here. Cloning is only relevant when contributing to the templates, not when using them.

Reply:
cloning or downloading doesnt matter, but cloning is more convenient, because you dont get a zip and the one command is also better then looking how to get the zip.